src/views/ProfileView.vue

```diff
@@ -12,8 +12,11 @@
   import { usePersonInfoStore, type PersonInfoStore, type PersonInfoResponse } from '@/stores/PersonInfoStore';
   import { usePersonenkontextStore, type Zuordnung, type PersonenkontextStore } from '@/stores/PersonenkontextStore';
   import { OrganisationsTyp } from '@/stores/OrganisationStore';
+  import { type RouteLocationNormalizedLoaded, useRoute } from 'vue-router';
 
-  type SchulDaten = {
+  const route: RouteLocationNormalizedLoaded = useRoute();
+
+  export type SchulDaten = {
     title: string;
     info?: string | null;
     schoolAdmins?: string[];
```

Comment:
```suggestion
    schulAdmins?: string[];
```
Die ganzen Entitäten (Schule, Klasse etc.) sollen immer deutsch benannt sein. Ist hier evtl. im letzten Review untergegangen.